docs/camino-messenger/bot/installation.mdx

```diff
@@ -32,6 +32,47 @@ software](#compiling-from-source) from its source code, while the second
 method [utilizes Docker](#building-docker-image) to create an image and run a
 container.
 
+## Known Issues
+
+The Camino Messenger Bot the following known issues should be considered:
+
+**Version: 10.2.0**
+
+- After start-up of a fresh bot installation takes 10 to 15 seconds before messages
+  can be sent or received. This period can be more depending on the network 
+  connections and if the bot receives messages before those 15 seconds.
+
+- Empty Message handling: In principle this is a bug in the Partner Plugin, 
+  an empty message should never be attempted to be sent to a partner. The
+  Partner Plugin should sanitize empty messages and we might implement validation
+  for empty messages once we start with the ProtoValidate project.
```

Comment:
This is hard to understand and confusing. It says that this is a bug in _Partner Plugin_ but then continues to say that an empty message should never be sent to the _Partner Plugin_. Which suggests that it's a bug with the bot. Needs clarification.